bin/cylc-check-software

```diff
@@ -188,9 +190,10 @@ def tex_module_search(tex_module, write=True):
     cmd = ['kpsewhich', '%s.sty' % tex_module]
     # This is less intensive & quicker than searching via 'find' or 'locate'.
     try:
-        process = Popen(cmd, stdin=open(os.devnull), stdout=PIPE)
+        process = pcylc(cmd, stdin=open(os.devnull), stdoutpipe=True)
         check_call(['test', '-n', process.communicate()[0].strip()],
                    stdin=open(os.devnull), stdout=PIPE, stderr=PIPE)
+
```

Comment:
Unnecessary blank line could be removed (not a hard rule, but we try to avoid spurious empty lines except where it is really helpful for readability).

Reply:
pep8 expects a two line gap here

Reply:
Does it really? (in the middle of an `try...except` block)?

Reply:
yep if I remove a line, E302 expected two blank lines found one. PeP8 likes two blank lines between function definitions

Reply:
ahh line numbers don't match! For me that's line 205! Changed...